howso/utilities/feature_attributes/base.py

```diff
@@ -780,10 +780,10 @@ def _process(self,  # noqa: C901
             except ImportError:
                 warnings.warn('Cannot infer extended nominals: not supported')
 
-        # Insert a ``sample`` for each feature, if possible.
+        # Insert a ``sample`` value (as string) for each feature, if possible.
         if include_sample:
             for feature_name in feature_attributes.keys():
-                sample = self._get_random_value(feature_name, no_nulls=True)
+                sample = str(self._get_random_value(feature_name, no_nulls=True))
```

Comment:
This will also serialize a None to "None".

Reply:
I don’t see the issue with that based on our discussion earlier.

Reply:
What if `"None"` is in the data itself? I think as is this is probably OK, but returning an actual null or not adding "sample" to the dict when there are no values in the column might be better.

Reply:
I've updated to it return None if it is indeed None. I like this because it will be obvious in the UI that there was no data to sample. Thank you.